### PR TITLE
Griefers 4 Pride Month

### PR DESCRIPTION
**Pull request by ToasterBiome**
# Document the changes in your pull request

https://github.com/yogstation13/Yogstation/assets/5091394/004d055f-df70-4b51-9cbb-e4e772d98a8a

# Changelog

:cl:  
rscadd: Griefers now support pride month!
/:cl:

### CODE REVIEW

**Comment by Hislyna** [comment]
Local HD increases plasmaflood by 42069%

**Comment by Moltijoe** [comment]
don't suppose you could use a holiday datum?

**Comment by ToasterBiome** [comment]
> don't suppose you could use a holiday datum?

i can

**Review by cuackles — APPROVED** [review]

File: code/modules/atmospherics/gasmixtures/gas_types.dm [hunk 1]
```diff
@@ -231,7 +231,12 @@ GLOBAL_LIST_INIT(nonreactive_gases, typecacheof(list(/datum/gas/oxygen, /datum/g
 	layer = FLY_LAYER
 	appearance_flags = TILE_BOUND
 
-/obj/effect/overlay/gas/New(state, alph)
+/obj/effect/overlay/gas/New(state, alph, gay)
 	. = ..()
 	icon_state = state
 	alpha = alph
+	if(gay)
+		icon_state = "pridesma"
```
Comment (alexkar598) [inline code/modules/atmospherics/gasmixtures/gas_types.dm]: All gasses become plasma when it's Gay?

Comment (ToasterBiome) [reply code/modules/atmospherics/gasmixtures/gas_types.dm]: it's only gay if the gas path is plasma and it's june

**Comment by Marmio64** [comment]
i feel like this might confuse people on what gas is in the air especially since this lasts for an entire month

**Review by adamsong — APPROVED** [review]
Coward making it only one week

**Comment by iamthesilly** [comment]
they put chemicals in the plasma to turn the atmos techs gay

**Pull request merged**